drivers/sensor/sht3xd/sht3xd.h

```diff
@@ -44,26 +45,24 @@
 #define SHT3XD_CLEAR_STATUS_WAIT_USEC   1000
 
 struct sht3xd_config {
-	char *bus_name;
+	const struct i2c_dt_spec bus;
```

Comment:
In general, specifying `const` qualifier to struct members should be avoided and the `const` qualification should be added during struct declaration instead (which, by the way, we already do).

Reply:
Fixed